packages/app/provider/font/index.tsx

```diff
@@ -1,9 +1,16 @@
 import React from 'react'
 import {
-  useFonts,
+  Raleway_300Light,
+  Raleway_400Regular,
+  Raleway_500Medium,
+  Raleway_600SemiBold,
+  Raleway_700Bold,
+} from '@expo-google-fonts/raleway'
+import {
   Inter_400Regular,
   Inter_600SemiBold,
   Inter_700Bold,
+  useFonts,
```

Comment:
we should import `useFonts` from the `expo-fonts` when we import more than 1 font.